src/main/features/dashboard/views/macro/claimStatus/statesPaidRejected.njk

```diff
@@ -40,19 +54,33 @@
   {% else %}
     {% set amount = claim.response.paymentDeclaration.paidAmount  %}
   {% endif %}
-  <h2 class="heading-small">{{ t('{{ claimantName }} rejected your admission of {{ amount }}', {
-      claimantName: claim.claimData.claimant.name,
-      amount: amount | numeral
-    }) }}</h2>
+  {% set mediation = claim.claimantResponse.freeMediation and claim.claimantResponse.freeMediation === YesNoOption.YES.option %}
+
+  {% if mediation %}
+    <h2 class="heading-small">{{ t('We’ll contact you with a mediation appointment') }}</h2>
+  {% else %}
+    <h2 class="heading-small">{{ t('The claimant has rejected your admission of {{ amount }}', { amount: amount | numeral}) }}</h2>
+  {% endif %}
+
   <p>{{ t('They said you didn’t pay them {{ amount }}.', { amount: amount | numeral }) }}</p>
-  <p>{{ t('You might have to go to a hearing. We’ll contact you if we set a hearing date to tell you how to prepare.') }}</p>
-  {{ _linkToDirectionsQuestionnairePage(claim.externalId) }}
-  <p>{{ t('Your defence won’t proceed if you don’t complete and return the form before 4pm on {{ dqDeadline }}.', {
-      dqDeadline: claim.directionsQuestionnaireDeadline | date
-    }) }}</p>
+
+  {% if mediation %}
+    {{ statesPaidRejectedMediationYESDefendantClaimDetails(claim) }}
+  {% else %}
+    {{ statesPaidRejectedMediationNODefendantClaimDetails(claim) }}
+  {% endif %}
+
   <p>{{ downloadResponseLink(claim) }}</p>
 {% endmacro %}
 
+{% macro statesPaidRejectedMediationYESDefendantClaimDetails(claim) %}
+  <p>{{ t('They agreed to try mediation. We’ll contact you with details of your appointment.') }}</p>
+{% endmacro %}
+
+{% macro statesPaidRejectedMediationNODefendantClaimDetails(claim) %}
+  <p>{{ t('You might have to go to a court hearing. We’ll contact you if we set a hearing date to tell you how to prepare.') }}</p>
```

Comment:
as above, this should say something about completing the paper DQs

Reply:
DQs content is being reviewed by Steve and I've been told to progress as is for now